pkg/server/node_tenant_test.go

```diff
@@ -45,7 +45,9 @@ func TestMaybeRedactRecording(t *testing.T) {
 			Add("tag_sensitive", tagSensitive).
 			Add("tag_not_sensitive", log.Safe(tagNotSensitive))
 		ctx := logtags.WithTags(context.Background(), tags)
-		ctx, sp := tracing.NewTracer().StartSpanCtx(ctx, "foo", tracing.WithForceRealSpan())
+		tracer := tracing.NewTracer()
+		tracer.SetRedactable(true)
```

Comment:
nit: slightly more "idiomatic" to set the span to redactable and not the tracer, right? This is closer to production usage. Not that it would make a difference.

Reply:
Done.